eth-reference-tests/src/test/java/tech/pegasys/artemis/reference/TestSuite.java

```diff
@@ -156,110 +153,64 @@ public static Object pathToObject(Path path, List<TestObject> testObjects) {
-      resultPair =
-          resultPair.stream()
-              .filter(
-                  pair -> {
-                    String walkPath = pair.getLeft();
-                    Boolean isSuccesTest =
-                        testSet.getFileNames().stream()
-                            .allMatch(fileName -> Files.exists(Path.of(walkPath, fileName)));
-                    pair.setValue(isSuccesTest);
-                    return isSuccesTest
-                        || testSetPostRemoved.getFileNames().stream()
-                            .allMatch(fileName -> Files.exists(Path.of(walkPath, fileName)));
-                  })
-              .collect(Collectors.toList());
-
-      List<Arguments> arguments = new ArrayList<>();
-      for (Pair<String, Boolean> pair : resultPair) {
-        String walkPath = pair.getLeft();
-        Boolean isSuccessTest = pair.getRight();
-        TestSet newTestSet;
-        if (isSuccessTest) {
-          newTestSet = new TestSet(testSet);
-        } else {
-          newTestSet = new TestSet(testSetPostRemoved);
-        }
-        List<Object> objects =
-            newTestSet.getFileNames().stream()
-                .flatMap(
-                    fileName -> {
-                      Bytes objectBytes = getSSZBytesFromPath(Path.of(walkPath, fileName));
-                      return newTestSet.getTestObjectByFileName(fileName).stream()
-                          .map(
-                              testObject ->
-                                  SimpleOffsetSerializer.deserialize(
-                                      objectBytes, testObject.getClassName()));
-                    })
-                .collect(Collectors.toList());
-        if (!isSuccessTest) {
-          objects.add(new BeaconState());
-        }
-        objects.add(isSuccessTest);
-        String filename = new File(walkPath).getName();
-        objects.add(filename);
-        arguments.add(Arguments.of(objects.toArray()));
-      }
-
-      Stream<Arguments> argumentsStream = arguments.stream();
-      return argumentsStream;
-    } catch (Exception e) {
-      LOG.log(Level.WARN, e.toString());
-    }
-    return null;
+  public static boolean isFilePathConfiguredForTest(TestSet testSet, String walkPath) {
+    boolean isIncludedPath =
+        testSet.getFileNames().stream()
+            .allMatch(fileName -> Files.exists(Path.of(walkPath, fileName)));
+    boolean isSuccessTest =
+        testSet.getFileNames().stream()
+                .filter(
+                    fileName ->
+                        fileName.contains("pre.yaml")
+                            || fileName.contains("pre.ssz")
+                            || fileName.contains("post.yaml")
+                            || fileName.contains("post.ssz"))
+                .collect(Collectors.toList())
+                .size()
+            > 1;
+    if (isSuccessTest) return isIncludedPath;
+    boolean isNotExcludedPath =
```

Comment:
I'm not sure what you're doing here

Reply:
I have two case. First case is we are doing a success test. In that case I need to return the result of `isIncludedPath`. The second case I still need that same value.